### DISCUSSION TITLE
Quais navegadores vocês estão usando?

### DISCUSSION THREAD

**Discussion by deppbrazil** [frontendbr/forum#1292] [Discussão]
# Fala Dev's blz? 😃  #

## Queria abrir aqui uma discussão rápida sobre qual ou quais navegadores vocês estão usando atualmente? ##

> Tenho utilizado muito nos últimos anos o [Chrome](https://www.google.com/chrome/) como principal tanto para desenvolvimento quanto para navegação, sei que come uma memória lascada mas tenho toda minha barra de favoritos e presets salvo nele... 😨 

### Fala ai o qual o de sua preferência, qual usam no dia-a-dia e porque? ### 

Entre os mais famosos do mercado **Chrome**, **Mozilla**, **Safari**, **Opera**, **IE**, **Edge**?
`(🔽  se tiver mais algum add ai 🔽)` 

___

* 🔗 Linkando uma issue relacionada porém focada em DevTools #432

**Comment by victorferreira** [comment]
- Safari como browser principal
- Chrome pra debugging no DevTools.

**Comment by onlurking** [comment]
[Firefox Developer Edition](https://www.mozilla.org/en-US/firefox/developer) como principal e para dev, e Chromium(como segundo browser no dev).

**Comment by luizbills** [comment]
Uso o Chrome. Recomendo você usar um gerenciador de abas, como o [Toby](https://chrome.google.com/webstore/detail/toby-for-chrome/hddnkoipeenegfoeaoibdmnaalmgkpip) caso você tenha pouca RAM ou até mesmo se você tiver muita RAM.

**Comment by rodrigooler** [comment]
Google Chrome.

**Comment by antoniotex** [comment]
> Uso o Chrome. Recomendo você usar um gerenciador de abas, como o [Toby](https://chrome.google.com/webstore/detail/toby-for-chrome/hddnkoipeenegfoeaoibdmnaalmgkpip) caso você tenha pouca RAM ou até mesmo se você tiver muita RAM.

Legal! Vou dar uma pesquisada nisso.

**Comment by fabiorocha** [comment]
Obrigado por linkar a issue sobre DevTools. Várias respostas lá que eu não tinha lido ainda. 😊

No trabalho uso quase todos, compilados na minha própria máquina (trabalho na implementação de browsers). Em casa uso uma combinação de Chrome + Firefox + Edge, na versão estável mais recente de cada um. Geralmente uso o browser de acordo com o caso de uso que preciso no momento - Velocidade / Streaming / Conservação de bateria / DevTools / etc. -- é interessante ver como cada browser tem pontos fortes e fracos, mas trabalhando nesse meio, tudo é relativo de onde os recursos são investidos durante cada release.

**Comment by willycamargo** [comment]
Tenho vários instalados para teste. Mas meu navegador do dia a dia (inclusive para dev) é o Chrome. Isso há mais de 3 anos já. Já tentei mudar, mas nenhum me convenceu.

**Comment by linconkusunoki** [comment]
**Uso pessoal**  
No celular eu uso o padrão que veio (da Xiaomi) gostei bastante dele.. mas acabo intercalando com o chrome por causa da barra de pesquisa que deixo na tela inicial.

**Trampo**
Chrome, to acostumado com o DevTools e não consegui acostumar com nenhum outro

**Comment by felipefialho** [comment]
- **Chrome** como navegador principal.
- **Canary** como navegador de desenvolvimento.

**Comment by ricalamino** [comment]
Ninguém usando o **[Brave](https://brave.com/)** ? Usei há um tempo, parecia bom.

**Comment by shadowmaru** [comment]
Firefox como browser principal. Firefox Developer Edition pra desenvolvimento.

No celular, Firefox e Firefox Focus.

**Comment by felipedotcom** [comment]
FireFox browser

Firefox Focus no celular

**Comment by felipecesr** [comment]
**Chrome** como navegador principal e desenvolvimento.

**Firefox**, **Edge** e **IE** para testes.

**Comment by andersoonweb** [comment]
Sempre usei muito o chrome para tudo!
Porém como todos sabem consome bastante RAM. Dessa vez, estou tentando utilizar o Firefox com seus addons. Estou adorando dá uma diferença legal no PC. mas... sinto bastante falta do chrome, principalmente quando vou abrir uma nova aba, haha.

**Comment by gustavofsantos** [comment]
Chrome dev ou Firefox dev. Mas também uso o BeakerBrowser pra testar as coisas que ando desenvolvendo com o Dat.

**Comment by caioincau** [comment]
Chrome para tudo, tentei usar o Safari como pessoal e o chrome para trabalho, mas não consigo me organizar com dois navegadores

**Comment by didiraja** [comment]
Chrome absolutamente, com adição do Opera para o popup de vídeos 😆

**Comment by vhfmag** [comment]
- Firefox Nightly como browser principal
- Chrome/Chromium pra usar Chromecast e Picture in picture, testar features novas e testar sites em outros browsers
- [Beaker](http://beakerbrowser.com/) de vez em quando

**Comment by felipefialho** [comment]
@vhfmag O que tá achando do Nightly?

**Comment by vhfmag** [comment]
@LFeh já uso há mais de um ano, curto bastante e em geral não tenho problemas com a estabilidade do nightly. Gosto do carregamento lazy de abas, das features de privacidade, da feature de mandar abas entre navegadores :heart: (mobile pra desktop e vice versa), do fato de que dá pra optar por abrir links em background no mobile e gostava do defunto suporte a RSS. Pontos negativos: muitas features chegam no Chrome primeiro e o Firefox tenta usar o tema do OS, o que leva a situações como essa:

![image](https://user-images.githubusercontent.com/357835/49166015-6b597180-f32a-11e8-8083-149329c6be39.png)
(dá pra ver que tem texto no campo de e-mail?)

**Comment by vitor-mendrone-deel** [comment]
Uso o Brave no dia a dia (Ads e tracker blocker por default) e percebi inclusive que isso reduziu meu uso de dados no celular. 

A diferença de tempo de carregamento em portais tipo o G1/Uol é gritante quando se bloqueia ads/trackers. Recomendo a todos!

**Comment by felipefialho** [comment]
@vhfmag @Mendrone Acho que vou dar uma testada no Brave e Nightly para o dia-a-dia heim

**Comment by rodrigooler** [comment]
@Mendrone por conta da indicação sua, eu acabei migrando para o Brave haha! <3 praticamente é um chrome que aparenta ser mais rápido e tem uns esquemas de adblock nativo.

**Comment by felipefialho** [comment]
O sync desses browsers (Brave e Nightly) entre dispositivos, funciona legal?

**Comment by vhfmag** [comment]
Sobre o Firefox Nightly, se cê tiver falando de sincronizar histórico, favoritos e senhas e formulários salvos, funciona muito bem! Dá até pra ver abas abertas nos seus outros dispositivos e mandar abas pra outros dispositivos.

**Comment by caioincau** [comment]
Estou testando o Brave e acho que eles não tem sync.

**Comment by vitor-mendrone-deel** [comment]
@rodrigooler É isso mesmo, ele usa a base do Chromium então é basicamente tudo igual

@LFeh @caioincau A parte de sync é o unico downside, acho que ele ainda não faz entre dispositivos, mas a economia de dados no celular e a performance acabam fazendo valer, nem que seja só pra navegar em sites de noticias, algo que eu faço muito

**Comment by vitor-mendrone-deel** [comment]
Alias, alguém aqui usou o Blisk? (https://blisk.io/) 

Ele se vende como um browser pra desenvolvedores, mas nunca testei. Acho que assim como o Brave, ele é baseado no Chromium

**Comment by linconkusunoki** [comment]
Vou testar este Brave também heauaehu
@Mendrone já usei o Blisk, é legal porque facilita visualizar de forma responsiva... mas ele não chega simular o device igual um Browserstack, é igual quando ativa o frame mobile do Chrome, então acabei deixando de lado

**Comment by heregoesnoth** [comment]
Não conhecia esse Brave e ele parece muito bom!

Em projetos uso o Firefox e pra uso pessoal o Chrome.

Vocês notam muita diferença nessas developer edition? (Firefox, Chrome etc) 
Baixei uma vez, mas não tive tempo de fuçar direito e nunca mais olhei.

@Mendrone já tinha ouvido falar e fui atrás essa semana, mas não tem versão pra Linux ainda :cry:

**Comment by mateusmiguel** [comment]
Às vezes eu me pego usando o Edge pra ver alguma bobagem na internet.  Não sei o que dizer quanto a isso.

**Comment by wkrueger** [comment]
Eu gosto muito do firefox, mas me perdoem - a dev tools do chrome é imbatível. O FF até ta tentando, mas não chega lá...

Então, tipo, não é lá legal ficar com 2 navegadores abertos, então melhor ficar só com a chrome botnet mesmo - mesmo que eu considere o FF melhor pra uso não-dev.

**Comment by matheusml** [comment]
#### Trabalho
Eu uso o Chrome pra trabalho, logado com a conta da minha empresa.

#### Pessoal
Uso o Brave e não consigo recomendar o bastante. É basicamente um Chrome modificado com foco em privacidade.

#### Smartphone
Brave também.

**Comment by rodrigooler** [comment]
Usei brave por pouco tempo nesses últimos dias, por conta que não tem o Sync ainda. Então acabei voltando pro Google Chrome mesmo.

**Comment by felipefialho** [comment]
O sync faz muita diferença pra mim. Esses navegadores tem essa funcionalidade em vista? @matheusml

**Comment by kakastefane** [comment]
Eu uso atualmente o Chrome como principal, já testei vários, mas o sync com a Google Account é maravilhoso e infelizmente não consegui me adaptar a outro browser por causa do sync...

**Comment by shadowmaru** [comment]
Sobre browsers, Google, e a notícia do Edge ir pro Chromium:

https://www.zeldman.com/2018/12/07/browser-diversity-starts-with-us

**Comment by matheusml** [comment]
@LFeh ainda não. Pra mim não ter _sync_ era um _deal-breaker_. Agora mudei de opinião, pra mim isso é uma **feature**.

**Comment by caioincau** [comment]
Eu tenho o mesmo feeling do @matheusml, me acostumei e até gosto de não ter mais urls poluídas no meu celular, como "localhost" ou urls internas da empresa.

**Comment by munizart** [comment]
Uso o firefox pra tudo, até no celular, pra mim não há outro navegador que bata

**Comment by claudiobernardo** [comment]
Chrome como principal.

**Comment by ocaioaugusto** [comment]
Caraca, eu to me sentindo excluído ahuauhauhuuhauha

Gosto de usar bastante o **Opera**, pra mim ele é bem funcional e vai direto ao ponto. Mas tô bem curioso em relação ao Brave, dá pra puxar as extensões do Chrome nele?
